raiden-ts/src/transfers/epics/retry.ts

```diff
@@ -60,7 +60,12 @@ const signedRetryMessage$ = (
         ),
       );
       // emit request once immediatelly, then wait until success, then retry every 30s
-      return retrySendUntil$(send, action$, notifier, httpTimeout);
+      return retrySendUntil$(
+        send,
+        action$,
+        notifier,
+        exponentialBackoff(pollingInterval, httpTimeout * 2),
```

Comment:
I'm a bit confused about this mix of the polling interval and the timeout for HTTP. It feels a bit randomly selected. Could we please document what each of the values is meant for in detail? Or point me to the existing one. Because right now it feels more like just using some values we already have disregarding their purpose.

Reply:
These values are just "constant" values which happen to be configurable, which enables us to set custom ones in tests to have these generic/general "waitings" & "timeouts" as sensible values which won't make our tests hang forever.
As I explained in the transport call, these are just config values, and we should keep in mind their defaults when using them:
- `pollingInterval=5s`, which is a nice initial retry delay: notice this is also the value used by `JsonRpcProvider` to poll for new blocks, so this means there's a good chance for, after this interval, the partner have retried, seen a new block/event, which makes them now accept a previously ignored message from us
- `httpTimeout=30s`, which is still a little bit spammy if the partner hasn't accepted our messages so far, probably they aren't going to, but we need to keep trying until something else happens (e.g. channel gets closed and disputed on-chain), so 60s is just a sensible value

Reply:
Hmm. I just want to increase the awareness how we use these values. In the end this are configurable values. If the user/developer can't know anymore (and **we too**) what is actually all affected by this, then we have a big problem. This example might sound awkward because we are not there yet with these values, but it shows the issue perfectly. So we have the option to configure the `settleTimeout`. The user of the SDK expects this value to be used for things that affect the settlement of channels. If we would then think, oh the default is actually a nice value we could use it also for something else and it is actually somehow related with some phantasy, then the user affects behavior with changes of this setting he does not intended. And this could lead to behavior also we don't understand anymore.

So as long as it makes sense to maintain such time values for a more general purpose I'm fine with it. Just make sure they are explained like that to the user. It must be possible to describe them. And when you use them you should check if you are still following this definition. Else this will be the start of a messy codebase.